enclave/core/calls.c

```diff
@@ -542,15 +548,15 @@ oe_result_t oe_ocall(uint16_t func, uint64_t arg_in, uint64_t* arg_out)
     /* If the enclave is in crashing/crashed status, new OCALL should fail
     immediately. */
     if (__oe_enclave_status != OE_OK)
-        OE_RAISE(__oe_enclave_status);
+        OE_RAISE_NO_TRACE((oe_result_t)__oe_enclave_status);
```

Comment:
There should probably be a comment explaining why this method has to use the `*_NO_TRACE` variants. I suspect that this is probably not the only method in calls.c where this is true (e.g. it's probably not a great idea to try and take locks and perform the ocalls that logging requires when you enter the enclave for the first time before it has finished initializing)

Reply:
Good point.

I added the following comment 
Given the logging implementation relies making an ocall to host, any 
** failures when handling oe_call should not invoke any oe_log() call for avoid
** never ending recursive calls. OE_RAISE and OE_CHECK macros contain call to 
** oe_log(), which involves a call to oe_ocall(). This is why they were replaced
** with OE_RAISE_NO_TRACE and OE_CHECK_NO_TRACE macros.